src/modules/SystemSettings.rb

```diff
@@ -329,8 +329,8 @@ def write_sysrq
       end
 
       log.info("Saving ENABLE_SYSRQ: #{enable_sysrq}")
-      sysctl_file.kernel_sysrq = enable_sysrq
-      sysctl_file.save
+      sysctl_config.kernel_sysrq = enable_sysrq
+      sysctl_config.save unless sysctl_config.conflict?
```

Comment:
Will it write something to user when conflict happen?

Reply:
Yes, he will get a popup with which file and tag it is conflicting.